src/OutputWriters/windowed_time_average.jl

```diff
@@ -321,13 +326,12 @@ function advance_time_average!(wta::WindowedTimeAverage, model)
 
         # Reset the "previous" interval time, subtracting a sliver that presents overshoot from accumulating.
         initialize_schedule!(wta.schedule, model.clock)
```

Comment:
This doesn't do anything anymore, right?

Reply:
Yes, you are correct!